### PR TITLE
Update Handbook for Changes to Checkout

### PR DESCRIPTION
**Pull request by Talishask**
The Checkout steps are being updated, following recommendations collected from the community during our [2022 Checkout Project](https://carpentries.org/blog/2022/09/checkout-project/). This pull request includes updates to the handbook pages to reflect an updated checkout process.

### CODE REVIEW

**Comment by Talishask** [comment]
### Summary of updates

**On the Cancellation & Make up page:**
- Add a missing space in the last paragraph

**Tracking Trainee Checkout Progress :**
- Deleted this page as it only included outdated information on the Core Team workflows for the administration and data entry. Any information regarding tracking checkout progress for trainees is included on the checkout page

**Trainers Guide:**
- Updated mention to tell trainee to prepare the entire lesson for the demo

**email_templates_trainers.md:**
- Updated a few sentences that mention to prepare the whole lesson.
- Updated a closing sentence that suggested that completing the demo completed checkout.

**email_templates_admin.md :**
- Updated email templates to remove outdated templates that referenced specific checkout steps and that are typically sent by the CT.
- Relocated a duplicate email template and labeled the duplicates as option 1 and option 2

File: topic_folders/instructor_training/email_templates_trainers.md [hunk 1]
```diff
@@ -78,11 +78,7 @@ Hi,
...
 
-I wanted to be sure you know that I may give you any segment of the lesson you prepared to teach, so you must be ready to teach any part of your chosen lesson. Some people prepare to teach only 5 minutes from a particular section and they often have to reschedule as they seldom are assigned the section they have prepared for. 
-
-A lesson corresponds to a single line in the lesson tables (https://software-carpentry.org/lessons/ , http://www.datacarpentry.org/lessons/ , and https://librarycarpentry.org/lessons/) and a single repository on GitHub. Some lessons have supplementary modules, but you do not need to be prepared to teach the supplementary modules for your teaching demonstration.
-
-For example, if you have chosen The Unix Shell, I may assign you any episode listed at http://swcarpentry.github.io/shell-novice/.
+Please be prepare to teach 1 section (episode) of your chosen Carpentries Lesson. For recommendedations on which lesson preparation, please see the page on [Suggested Lessons for Teaching Demonstrations](https://carpentries.github.io/instructor-training/demo_lessons).
```
Comment (karenword) [inline topic_folders/instructor_training/email_templates_trainers.md:81]:
```suggestion
Please be prepared to teach for 5 minutes, beginning at the start of your chosen episode from a Carpentries lesson. Please review the page on [Suggested Lessons for Teaching Demonstrations](https://carpentries.github.io/instructor-training/demo_lessons) to ensure that your selection is on the list.
```

File: topic_folders/instructor_training/email_templates_trainers.md [hunk 2]
```diff
@@ -127,7 +123,7 @@ I’m happy to say that you have passed your teaching demonstration! You demonst
 
 I’ve forwarded this information to our Core Team. If this was the last step in your Instructor Training checkout, you should hear from them within about a week. 
 
-Welcome to The Carpentries Instructor community!
+Looking forward to having you as a Carpentries Instructor!
```
Comment (karenword) [inline topic_folders/instructor_training/email_templates_trainers.md:126]: I have been trying to reduce words like "looking" where reasonable alternatives are less ableist. In this case I think the sentence can be deleted or left as-is, assuming Trainers will customize anyway.
```suggestion

```

File: topic_folders/instructor_training/trainers_guide.md [hunk 3]
```diff
@@ -25,12 +25,11 @@ If you would like to watch an example teaching demo, there is a recording of one
...
 - Get acquainted with using [Zoom](../communications/tools/zoom_rooms.md) for videoconferencing.
-- A day or two before the demo, send a reminder email using the [email template](https://docs.carpentries.org/topic_folders/instructor_training/email_templates_trainers.html#reminder-teaching-demo). This often leads to people letting you know they cannot make it, or clarifications regarding what to prepare for (episode versus lesson)
-- For each trainee, pick a suitable starting point in the lesson that they have chosen. Suggested lessons are available in the [Instructor Training Curriculum](https://carpentries.github.io/instructor-training/demo_lessons) (under *More*). Do not have them start in the middle of an episode. Note that some lessons (e.g., the Software Carpentry R lesson using inflammation data) have supplementary episodes. Do not pick from those.  
+- A day or two before the demo, send a reminder email using the [email template](https://docs.carpentries.org/topic_folders/instructor_training/email_templates_trainers.html#reminder-teaching-demo). This often leads to people letting you know they cannot make it, or clarifications regarding how to prepare for the demo.
```
Comment (karenword) [inline topic_folders/instructor_training/trainers_guide.md:28]:
```suggestion
- A day or two before the demo, send a reminder email using the [email template](https://docs.carpentries.org/topic_folders/instructor_training/email_templates_trainers.html#reminder-teaching-demo). This often leads to people letting you know they cannot make it, or clarifications regarding how to prepare for the demo.  
- For each trainee, click the link they have provided for their episode and ensure that it is on the [list of suggested lessons](https://carpentries.github.io/instructor-training/demo_lessons)
```

File: topic_folders/instructor_training/email_templates_trainers.md (hunk 2 above)
Comment (karenword) [inline topic_folders/instructor_training/email_templates_trainers.md:126]:
```suggestion
Congratulations!
```

**Comment by karenword** [comment]
I have completed review, merged suggestions, and resolved conflicts. If any further updates are made, please check for conflicts so this is ready for a hassle-free merge on 13 August.

**Pull request merged**